src/SystemPaths.cc

```diff
@@ -403,6 +390,8 @@ std::string SystemPaths::FindFile(const std::string &_filename,
   // Try appending to local paths
   else
   {
+    ignerr << "Here: " << cwd() << " " << filename << std::endl;
```

Comment:
remove this debugging message

Reply:
this one too?

Reply:
removed in https://github.com/ignitionrobotics/ign-common/pull/238/commits/6ab5b607bd7d36acea1fec722965a13bb1d507dc